pdc_client/__init__.py

```diff
@@ -135,7 +135,7 @@ def __setattr__(self, name, value):
         return super(_SetAttributeWrapper, self).__setattr__(name, value)
 
 
-class PDCClient(_SetAttributeWrapper):
```

Comment:
But removing this functionality breaks API.

From commit description I don't see what the issue is. What PDCClient attributes cannot be set? Can you write a test case for it in "tests/api/tests.py"?

Reply:
I agree that it would be nice to have a test for this.

Reply:
Yes, like @lubomir mentioned, the reason why it raise "404 error" is there is a self.page set after the super call.
And I think remove this don't break the API, because in previous codes, we can support codes like "client.aaa".
@hluk and @lubomir  I can add the test on codes, good idea.

Reply:
@lubomir and @hluk I have added the test case.
change the port to 8278, as it raises "socket.error: Address already in use".

@hluk  I think we should still keep this _SetAttributeWrapper, because we should set the "self.client = client" and "self.page_size = page_size" in wrapper class then we could re-write the __setattr__,